Source/Core/Core/PowerPC/MMU.cpp

```diff
@@ -1116,6 +1116,26 @@ static TranslateAddressResult TranslatePageAddress(const u32 address, const XChe
   return TranslateAddressResult{TranslateAddressResult::PAGE_FAULT, 0};
 }
 
+static bool overlaps_memcheck(u32 pageEndAddress)
+{
+  if (!memchecks.HasAny())
+    return false;
+  u32 page_end_suffix = ((1 << BAT_INDEX_SHIFT)) - 1;
+  for (TMemCheck memcheck : memchecks.GetMemChecks())
+  {
+    u32 memcheck_region_page_start = 0x00FFFFFF & memcheck.start_address;
```

Comment:
I noticed the addresses given when entering this doesn't start with 0x80 while the memcheck addresses does.  This removes the 0x80 on the memchecks addresses.